pkg/services/ngalert/notifier/channels/slack.go

```diff
@@ -254,7 +252,7 @@ func (sn *SlackNotifier) buildSlackMessage(ctx context.Context, as []*types.Aler
 			{
 				Color:      getAlertStatusColor(alerts.Status()),
 				Title:      tmpl(sn.Title),
-				Fallback:   tmpl(sn.Fallback),
+				Fallback:   tmpl(sn.Title),
```

Comment:
Is this intentional? Why?

Reply:
The content of title and fallback was same. So instead of giving configuration option for both, I thought of reducing the options and re-use the title.